ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -3309,6 +3309,7 @@ tests:
   steps:
     cluster_profile: nutanix-qe
     env:
+      API_CALL_TIMEOUT: "30"
```

Comment:
shall we also update the 4.20 ones ?

Reply:
Sure, updated 4.20

Reply:
we also need to update 4.19 z-stream upgrade ci jobs.

Reply:
I only increased timeout in multi-nic jobs, it's TP feature so no upgrade jobs